packages/cli/src/util/client.ts

```diff
@@ -93,7 +93,8 @@ export default class Client extends EventEmitter implements Stdio {
       : '';
 
     if (opts.accountId || opts.useCurrentTeam !== false) {
-      const query = new URLSearchParams(parsedUrl.query);
+      // TODO - not sure what the correct TypeScript fix is here
+      const query = new URLSearchParams(parsedUrl.query as any);
```

Comment:
This might be a real bug if the constructor doesn’t handle the case with multiple keys. 

Might be better to change this to “new URL(_url)”

Reply:
The actual TS error here is: 

```
Argument of type 'ParsedUrlQuery' is not assignable to parameter of type 'string | URLSearchParams | Record<string, string | readonly string[]> | Iterable<[string, string]> | readonly [string, string][] | undefined'.
  Type 'Dict<string | string[]>' is missing the following properties from type 'readonly [string, string][]': length, concat, join, slice, and 19 more.
```

I believe `ParsedUrlQuery` is an alias for `Dict<string | string[]>`, which should satisfy the `Record<string | readonly string[]>` option but doesn't. 

Here is some type definitions from `@types/node`:
```
interface ParsedUrlQuery extends NodeJS.Dict<string | string[]> { }
interface Dict<T> {
  [key: string]: T | undefined;
}
```